testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -1375,7 +1375,7 @@ public void testBasicSearch() throws IOException {
             hostWithPort
```

Comment:
can we add two test cases for edge conditions:
1. Validate that it returns an error when page-size > upper bound
2. Validate that it returns an error when page-size < lower bound